frontend/src/app/(navfooter)/settings/editor/EditorSettings.tsx

```diff
@@ -32,5 +54,27 @@ export default function EditorSettings() {
                 </div>
             </Checkbox>
         </Section>
+        <Section title="Language server">
+            <Checkbox
+                checked={languageServerEnabled}
+                onChange={state => {
+                    setLanguageServerEnabled(state)
+                    setLanguageServerSettingsMutated(true)
```

Comment:
Perhaps this could be set with an effect?

Reply:
I think that could run into the same problems with running on mount, but I was able to use an effect to make things read less clunkily.